src/jdk.compiler/share/classes/com/sun/tools/javac/file/JavacFileManager.java

```diff
@@ -193,7 +193,7 @@ private Path getPath(String first, String... more) {
     }
 
     /**
-     * Set whether or not to use ct.sym as an alternate to rt.jar.
+     * Set whether or not to use ct.sym as an alternate
```

Comment:
As an alternate to what? This needs something else.

Reply:
should "to the image modules files"  be used instead ?